assets/scss/base/layouts/_layouts.scss

```diff
@@ -96,3 +96,47 @@ body {
 .row {
   margin-bottom: 20px;
 }
+
+/*
+Half column container component
+
+Experimental: Half column component
+
+Markup:
+<div class="column-half-container">
+    <div class="grid-wrapper row-border">
+        <div class="grid grid-1-2">
+            <div>Some content</div>
+        </div>
+        <div class="grid grid-1-2">
+            <div>Some content</a>
+        </div>
+    </div>
+</div>
+
+.column-half-container - component containing two half column grids redefining parts of row-border and grid-wrapper elements
+*/
+.column-half-container {
```

Comment:
Can this not use https://github.com/hmrc/assets-frontend/blob/master/assets/scss/base/_grid.scss#L69 ?